src/epitools/_sample_data.py

```diff
@@ -10,11 +10,72 @@
 
 from pathlib import Path
 
-from skimage.io import imread
+import numpy as np
+import pandas as pd
+
+import epitools._reader
 
 
 def load_sample_data():
```

Comment:
Type the output

Reply:
I've added types to all the functions and tests. I was having issues with mypy complaining about `tuple` not being subscriptable on Python 3.8 (even after import annotation from `__future__`), so I'm using e.g. `typing.Tuple`